thorlcr/activities/keydiff/thkeydiff.cpp

```diff
@@ -159,7 +159,7 @@ class CKeyDiffMaster : public CMasterActivity
     {
         Owned<IWorkUnit> wu = &container.queryJob().queryWorkUnit().lock();
         Owned<IWUResult> r = wu->updateResultBySequence(helper->getSequence());
-        r->setResultStatus(ResultStatusCalculated);
+        //Do not mark the result as calculated - because the patch file isn't a valid result
```

Comment:
I presume you know more about what this means (to be unset) than me..
e.g. if you were to query the Result Status .. when it has not been set..